### PR TITLE
Add Blackmagic devices

### PR DESCRIPTION
**Pull request by Ivarrrrr**
Add Blackmagic devices

### CODE REVIEW

File: device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+manufacturer: Blackmagicdesign
```
Comment (sjurtf) [inline device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml:2]: I suggest we name this manufacturer `Blackmagic Design` instead

File: device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+model: ATEM 2ME Production Studio 4K
+slug: blackmagicdesign-atem-2me-production-studio-4k
```
Comment (sjurtf) [inline device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml:4]: `blackmagic-design`

**Review by sjurtf — CHANGES_REQUESTED** [review]

File: device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml (hunk 1 above)
Comment (Ivarrrrr) [reply device-types/Blackmagicdesign/ATEM-2ME-Production-Studio-4K.yaml:2]: Jeg er egentlig enig, men dette formatet var det netbox-device-greiene ville ha.

**Review by sjurtf — CHANGES_REQUESTED** [review]
Please also change the name of the manufacturer folder.

**Pull request closed without merging**